gui/wxpython/lmgr/toolbars.py

```diff
@@ -235,10 +330,22 @@ def _toolbarData(self):
 
         return self._getToolbarData(
             (
-                ("nvizCmd", icons["cmd"], self.OnNvizCmd),
+                (
+                    ("nvizCmd", _("Generate command for m.nviz.image")),
+                    icons["cmd"],
+                    self.OnNvizCmd,
+                ),
                 (None,),
-                ("settings", icons["settings"], self.parent.OnNvizPreferences),
-                ("help", icons["help"], self.OnHelp),
+                (
+                    ("settings", _("3D view settings")),
+                    icons["settings"],
+                    self.parent.OnNvizPreferences,
+                ),
+                (
+                    ("help", _("3D view help")),
```

Comment:
Just for consistency, shouldn't this one be "Show manual" too as in all other help icons?

Reply:
![main_win_3d_view_toolbar](https://user-images.githubusercontent.com/50632337/151561042-053b0541-9185-4740-9c42-80a1d3bf5f4f.png)

Reply:
For the record, I think the long-term goal should be to move the ones for 3D somewhere else (3D tab, map display, but I don't have concrete suggestion right now.